### PR TITLE
DIT-1818: Add search cases button in liabilities case header

### PR DESCRIPTION
**Pull request by frikit**

### CODE REVIEW

File: app/models/viewmodels/LiabilityViewModel.scala [hunk 1]
```diff
@@ -16,29 +16,40 @@
...
-    isNewCase && hasPermissions
-  }
+  def isNewCase: Boolean = caseStatus == CaseStatus.NEW
+
+  def showRulingAndKeywordsTabs: Boolean =
+    caseStatus == CaseStatus.OPEN ||
+      caseStatus == CaseStatus.REFERRED ||
+      caseStatus == CaseStatus.REJECTED ||
+      caseStatus == CaseStatus.SUSPENDED ||
+      caseStatus == CaseStatus.COMPLETED
```
Comment (DavidGregory084) [inline app/models/viewmodels/LiabilityViewModel.scala:38]: Maybe we could use `Set(CaseStatus.OPEN, CaseStatus.REFERRED, /* etc */).contains(caseStatus)` here?

Comment (frikit) [reply app/models/viewmodels/LiabilityViewModel.scala:38]: If you think it does make big difference I can make this change, to me there is no difference

Comment (frikit) [reply app/models/viewmodels/LiabilityViewModel.scala]: done :)

Comment (DavidGregory084) [reply app/models/viewmodels/LiabilityViewModel.scala]: Haha I'm not sure it makes a big difference either. :) I have a distant memory that we already have Sets of statuses somewhere in the code.

**Pull request merged**